static/js/components/WidgetPrefsDialog.jsx

```diff
@@ -63,38 +65,82 @@ const WidgetPrefsDialog = ({
       <Dialog open={open} onClose={handleClose} style={{ position: "fixed" }}>
         <DialogTitle>{title}</DialogTitle>
         <DialogContent>
-          <form noValidate autoComplete="off">
-            {Object.keys(formValues).map((key) => (
-              <div key={key}>
-                <TextField
-                  id={`${key}`}
-                  size="small"
-                  label={key}
-                  value={state[key]}
-                  onChange={handleInputChange}
-                  variant="outlined"
-                />
-              </div>
-            ))}
+          <form
+            noValidate
+            autoComplete="off"
+            onSubmit={handleSubmit(formSubmit)}
+          >
+            {Object.keys(initialValues).map((key) => {
+              if (
+                typeof initialValues[key] === "object" &&
+                initialValues[key].constructor === Object
+              ) {
+                return (
+                  <div key={key} className={classes.inputSectionDiv}>
+                    Select {key}:
```

Comment:
Maybe wrap this in a Typography tag? Maybe variant="subtitle2" or something like that. I think it looks a little off right now because the header here is a smaller font than the checkbox options.

Reply:
@kmshin1397 done: 
![image](https://user-images.githubusercontent.com/7230285/102804463-aa721980-436e-11eb-9b5b-151014c63ad1.png)